### ISSUE TITLE
[개선제안] 번역 모범 사례 Wiki 내 링크 업데이트

### ISSUE THREAD

**Issue by boyeonihn** [javascript-tutorial/ko.javascript.info#1429]
## 자바스크립트 튜토리얼 한국 저장소 개선을 위해 제안할 사항
오픈소스 기여 준비를 위해 번역 모범 사례 위키를 읽다가 링크 몇 개가 업데이트가 필요하다는 걸 발견했습니다. 링크 클릭을 했더니 이런 화면이 보입니다: 
![errorPage](https://user-images.githubusercontent.com/95451902/180605773-4aca0e57-7a69-487f-9ffc-41f08d73b090.png)

개선을 위해 제안할 사항은 세 가지입니다: 
1. 쌍점에 관한 참고 링크는 [이 링크](https://kornorms.korean.go.kr//regltn/regltnView.do#a753)로 변경되는 것을 제안합니다 
2. 쉼표에 관한 참고 링크는 [이 링크](https://kornorms.korean.go.kr//regltn/regltnView.do#a732)로 변경되는 것을 제안합니다
3. 짝을 이루는 단어를 이을 때는 가운뎃점 사용하기의 [맞춤법 관례](https://www.korean.go.kr/front/page/pageView.do?page_id=P000196&mn_id=30) 링크를 이 [링크](https://kornorms.korean.go.kr/regltn/regltnView.do?regltn_code=0001&regltn_no=714#a749)로 변경되는 것을 제안합니다

## 제안한 이유
처음으로 번역 오픈소스 기여를 준비하시는 분들이 올바르고 정확한 번역 작업을 진행하실 수 있게 참고 링크가 잘 작동해야  한다고 생각하기 때문에 제안합니다! 

## 추가적으로 논의할 사항
현재로서는 없습니다. 번역 모범 사례 위키 페이지가 정리를 깔끔하게 해줘서 정확한 번역에 대한 이해가 넓어졌습니다. 감사합니다! :)

**Comment by Violet-Bora-Lee** [comment]
오오 deprecated된 링크를 찾아주셔서 감사합니다.
개선 내용을 담은 Pull Request를 만들어주실수 있으실까요?

**Comment by boyeonihn** [comment]
보라님 안녕하세요! 

제가 공부/검색을 좀 해봤는데, wiki 내용 자체는 코드소스가 담긴 저장소에 위치하지 않았기 때문에 PR를 하기가 어려운 걸로 알고 있습니다. 또한 upstream 저장소를 포크해도 wiki파일은 같이 포크가 되지 않더라구요!

최선의 방법은 현재 [이 방법](https://gist.github.com/larrybotha/10650410) 밖에 찾지 못해서, 일단 Wiki.git 링크를 이용해 수동으로 Wiki 내용을 내려받고 업데이트 한 다음 변경이력이 담긴 제 개인 wiki.git 링크를 보라님께 공유해드리는 것 밖에 없는 것 같아요. 

링크가 업데이트 된, 현재 제 fork된 저장소 Wiki에 위치한 모범 번역 사례 md 파일: [링크](https://github.com/boyeonihn/ko.javascript.info/wiki/%EB%B2%88%EC%97%AD-%EB%AA%A8%EB%B2%94-%EC%82%AC%EB%A1%80)

업데이트 된 부분: 쌍점 링크, 쉼표 링크, 그리고 "짝을 이루는 단어를 이을 때는 가운뎃점 사용하기" 링크. 

1. 더 현명한 방법으로 실제로 PR를 할 수 있는 방법을 아신다면 알려주시면 감사하겠습니다 (제가 한 30-40분 찾았는데 직접적인 Wiki PR는 어렵다는 말만 하더라구요) 
2. 만약에 현재로썬 이 방법 밖에 없다면, 링크 업데이트가 된 제 wiki.git 페이지 한 번 확인해보신 후 ok 주시면 제가 직접 ko.javascript.info 위키 페이지 링크를 업데이트하겠습니다. 

감사해요!

**Comment by Violet-Bora-Lee** [comment]
안녕하세요, 2번으로 문서업데이트 부탁드릴게요.

**Comment by boyeonihn** [comment]
네 보라님 제가 [개인 저장소에 올린 업데이트 된 Wiki 문서](https://github.com/boyeonihn/ko.javascript.info/wiki/%EB%B2%88%EC%97%AD-%EB%AA%A8%EB%B2%94-%EC%82%AC%EB%A1%80) 를 올렸으니 한 번 확인해주세요:
